running locally/kafka/readme.md

```diff
@@ -0,0 +1,499 @@
+ _key              | varchar |       | Key text
+ _key_corrupt      | boolean |       | Key data is corrupt
+ _key_length       | bigint  |       | Total number of key bytes
+ _message          | varchar |       | Message text
+ _message_corrupt  | boolean |       | Message data is corrupt
+ _message_length   | bigint  |       | Total number of message bytes
+(21 rows)
+```
+```
+presto:tpch> SELECT * FROM customer LIMIT 5;
+ kafka_key | row_number | customer_key |        name        |                address                | nation_key |      phone      | account_balance | market_segment |                                                      comment
+-----------+------------+--------------+--------------------+---------------------------------------+------------+-----------------+-----------------+----------------+---------------------------------------------------------------------------------------------------------
+         1 |          2 |            2 | Customer#000000002 | XSTf4,NCwDVaWNe6tEgvwfmRchLXak        |         13 | 23-768-687-3665 |          121.65 | AUTOMOBILE     | l accounts. blithely ironic theodolites integrate boldly: caref
+         3 |          4 |            4 | Customer#000000004 | XxVSJsLAGtn                           |          4 | 14-128-190-5944 |         2866.83 | MACHINERY      |  requests. final, regular ideas sleep final accou
+         5 |          6 |            6 | Customer#000000006 | sKZz0CsnMD7mp4Xd0YrBvx,LREYKUWAh yVn  |         20 | 30-114-968-4951 |         7638.57 | AUTOMOBILE     | tions. even deposits boost according to the slyly bold packages. final accounts cajole requests. furious
+         7 |          8 |            8 | Customer#000000008 | I0B10bB0AymmC, 0PrRYBCP1yGJ8xcBPmWhl5 |         17 | 27-147-574-9335 |         6819.74 | BUILDING       | among the slyly regular theodolites kindle blithely courts. carefully even theodolites haggle slyly alon
+         9 |         10 |           10 | Customer#000000010 | 6LrEaV6KR6PLVcgl2ArL Q3rqzLzcT1 v2    |          5 | 15-741-346-9870 |         2753.54 | HOUSEHOLD      | es regular deposits haggle. fur
+(5 rows)
+```
+```
+presto:tpch> SELECT sum(account_balance) FROM customer LIMIT 10;
+   _col0
+------------
+ 6681865.59
+(1 row)
+```
+Now all the fields from the customer topic messages are available as Presto table columns.
+
+## Step 7: Use live data
+Presto can query live data in Kafka as it arrives. To simulate a live feed of data, this tutorial sets up a feed of live tweets into Kafka.
+
+## Setup a live Twitter feed
+Download the twistr tool
+```
+$ curl -o twistr https://repo1.maven.org/maven2/de/softwareforge/twistr_kafka_0811/1.2/twistr_kafka_0811-1.2.sh
+$ chmod 755 twistr
+```
+Create a developer account at https://dev.twitter.com/ and set up an access and consumer token.
+
+Create a twistr.properties file and put the access and consumer key and secrets into it:
```

Comment:
This is a general comment. You describe the action "create a file with the following content" three different ways in this document that I can find: 

1. add a catalog properties file etc/catalog/kafka.properties for the Kafka connector. This file lists the Kafka nodes and topics:

2. Add the following file as etc/kafka/tpch.customer.json and restart Presto:

3. Create a twistr.properties file and put the access and consumer key and secrets into it:

Users benefit from consistent phrasing for the same action or object. Think about revising these three instructions to use the same words.